src/style.css

```
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  background-color: black;
+}
+
+.blue-box {
+  width: 348px;
```

```suggestion
  width: 350px;
```
и задай `box-sizing: border-box` (почитай об этом свойстве в гугле)